__snapshots__/Switch.md

```diff
@@ -3,7 +3,13 @@
 #### `loads`
 
 ```html
-<input id="input" tabindex="0" type="checkbox" />
+<input
+    aria-checked="false"
+    id="input"
+    role="switch"
+    tabindex="0"
```

Comment:
input type=checkbox shouldn't need tabindex to be set as it is natively focusable

Reply:
This is a part of our work around for a lack of broad support for `delegatesFocus` in shadow DOM and is expected.

Reply:
ok - its a no-op so no objection